### PR TITLE
SAK-41337 send the current instructor over to Turnitin when possible

### PR DESCRIPTION
**Pull request by ottenhoff**

### CODE REVIEW

File: content-review/impl/turnitin/src/main/java/org/sakaiproject/contentreview/turnitin/TurnitinReviewServiceImpl.java [hunk 1]
```diff
@@ -371,7 +371,7 @@ public String getReviewReportInstructor(String contentId, String assignmentRef,
 		Map params = TurnitinAPIUtil.packMap(turnitinConn.getBaseTIIOptions(), "fid", fid, "fcmd", fcmd, "assignid",
 				assignid, "cid", cid, "oid", oid, "utp", utp);
 
-		params.putAll(getInstructorInfo(item.getSiteId()));
+		params.putAll(getInstructorInfo(item.getSiteId(), false, userId));
```
Comment (ottenhoff) [inline content-review/impl/turnitin/src/main/java/org/sakaiproject/contentreview/turnitin/TurnitinReviewServiceImpl.java]: This is the big change ..... old call would have returned the first instructor in the site ......

Untested .....

**Comment by baholladay** [comment]
Why was this closed instead of merged?

**Comment by ottenhoff** [comment]
Because it didn't work

**Comment by baholladay** [comment]
That's as good of a reason that I can think of! Thanks for the PR to improve our integration, I had it on my list to test.

**Comment by ottenhoff** [comment]
I don't know what the right approach is here. Re-adding the TII Roster Sync Job .... or fixing the instructor code to add the instructor user account to TII before attempting to view the report.... thoughts?

**Comment by baholladay** [comment]
I'm not as familiar with the current TII integration. Your approach seemed correct, where you try to find the specific requesting instructor, instead of the "first". Why did this not work?

**Comment by ottenhoff** [comment]
Because TII apparently requires the user to be created first .... user creation is a separate process.

**Pull request closed without merging**